lms/static/js/student_account/views/LoginView.js

```diff
@@ -1,131 +1,125 @@
-var edx = edx || {};
-
-(function($, _, gettext) {
+;(function (define) {
     'use strict';
-
-    edx.student = edx.student || {};
-    edx.student.account = edx.student.account || {};
-
-    edx.student.account.LoginView = edx.student.account.FormView.extend({
-        el: '#login-form',
-
-        tpl: '#login-tpl',
-
-        events: {
-            'click .js-login': 'submitForm',
-            'click .forgot-password': 'forgotPassword',
-            'click .login-provider': 'thirdPartyAuth'
-        },
-
-        formType: 'login',
-
-        requiredStr: '',
-
-        submitButton: '.js-login',
-
-        preRender: function( data ) {
-            this.providers = data.thirdPartyAuth.providers || [];
-            this.hasSecondaryProviders = (
-                data.thirdPartyAuth.secondaryProviders && data.thirdPartyAuth.secondaryProviders.length
-            );
-            this.currentProvider = data.thirdPartyAuth.currentProvider || '';
-            this.errorMessage = data.thirdPartyAuth.errorMessage || '';
-            this.platformName = data.platformName;
-            this.resetModel = data.resetModel;
-
-            this.listenTo( this.model, 'sync', this.saveSuccess );
-            this.listenTo( this.resetModel, 'sync', this.resetEmail );
-        },
-
-        render: function( html ) {
-            var fields = html || '';
-
-            $(this.el).html( _.template( this.tpl, {
-                // We pass the context object to the template so that
-                // we can perform variable interpolation using sprintf
-                context: {
-                    fields: fields,
-                    currentProvider: this.currentProvider,
-                    errorMessage: this.errorMessage,
-                    providers: this.providers,
-                    hasSecondaryProviders: this.hasSecondaryProviders,
-                    platformName: this.platformName
+    define(['jquery', 'underscore', 'js/student_account/views/FormView'], function($, _, FormView) {
+
+        var LoginView = FormView.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the view, i.e.
```
return FormView.extend({
```